### DISCUSSION TITLE
How do I remove all templates from a wikicode object?

### DISCUSSION THREAD

**Discussion by dpriskorn** [earwig/mwparserfromhell#299] [Q&A] [answered]
Could you add an example of how to use the remove method to the docs?

**Comment by dpriskorn** [comment]
I found the strip_code method which seems to do what I want. Please add it to the documentation 😀

**Comment by earwig** [comment, marked as answer]
It’s already documented: https://mwparserfromhell.readthedocs.io/en/latest/api/mwparserfromhell.html#mwparserfromhell.wikicode.Wikicode.strip_code
